src/core/client_channel/retry_throttle.cc

```diff
@@ -44,26 +60,24 @@ ServerRetryThrottleData::ServerRetryThrottleData(
   // we will start out doing the same thing on the new one.
   if (old_throttle_data != nullptr) {
     double token_fraction =
-        static_cast<uintptr_t>(
-            gpr_atm_acq_load(&old_throttle_data->milli_tokens_)) /
+        static_cast<double>(
```

Comment:
Is this `static_cast<>` actually needed?

Reply:
I'd prefer to keep this one for clarity